net/netmon/netmon.go

```diff
@@ -401,7 +409,7 @@ func (m *Monitor) handlePotentialChange(newState *interfaces.State, forceCallbac
 
 	delta.Major = m.IsMajorChangeFrom(oldState, newState)
 	if delta.Major {
```

Comment:
I'm trying to get rid of this "Major" bool (it's so undefined).

Does this need to depend on "Major"? Can it not? :)

Reply:
@bradfitz - I was going off `Major` because that's what the `gwValid` boolean does. I can have it re-log on every link change, but that seems super noisy ... maybe just log when it changes, in all cases?

Reply:
Updated; PTAL?